### DISCUSSION TITLE
Xdrip notifications on Garmin Venu watch

### DISCUSSION THREAD

**Discussion by pawe70** [NightscoutFoundation/xDrip#3410] [Q&A]
Hi, i am running xdrip on a samsung galaxy and have a garmin venu watch connected, several times during the week notifications stops on the watch, i then have to open garmin connect app and enable notification again, this problem only happens with xdrip. Please help

**Comment by Navid200** [comment]
This is WearOS, right?

**Reply by pawe70** [reply]
I think it is GarminOS, but not 100% sure

**Comment by PhilTU** [comment]
Do you have a glucose value on the watch https://apps.garmin.com/search?keywords=cgm+glucose or just using notifications ?

**Reply by pawe70** [reply]
Hi, i am using this watchface https://apps.garmin.com/apps/8fab3746-f56f-4b41-b3c7-5f4aaeaed6c9

**Reply by PhilTU** [reply]
Does it show glucose values?

Troubleshooting and setup at https://github.com/swissalpine/GARMIN-xDrip-Spike-Nightscout-Watchface/

**Reply by PhilTU** [reply]
https://github.com/swissalpine/GARMIN-xDrip-Spike-Nightscout-Watchface/?tab=readme-ov-file#troubleshooting-qa

**Reply by pawe70** [reply]
All of this is ok, what happens is that notifications keeps disable in the Garmin Connect app several times during the week, i then have to manually enable it again

![Notifications](https://github.com/NightscoutFoundation/xDrip/assets/110022710/a34f818d-3d7d-43c6-a294-c02bfa26bb53)

**Reply by PhilTU** [reply]
Probably do better in a Garmin forum or on the Swissalpine github "How do I stop notifications disabling regularly"

As xDrip+ is providing notifications correctly ?
